### PR TITLE
[6]  Product Page using HTML

### PR DESCRIPTION
**Pull request by ElJentle**
## Task: `Product Page using HTML`
- Added html task

Foodcoonline Home Page  3
Foodcoonline Home Page  4
Foodcoonline Home Page  5
Foodcoonline Home Page  6
Foodcoonline Home Page  7
Foodcoonline Home Page  8

Foodcoonline Home Page  1
Foodcoonline Home Page  2

### CODE REVIEW

File: submissions/emmanuelEsuong/introductionToHtml/index.html [hunk 1]
```diff
@@ -0,0 +1,214 @@
+<<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/emmanuelEsuong/introductionToHtml/index.html:1]: Well done on the job done so far.
You need to remove all styles in this file. Check criteria 1 for this task.

File: submissions/emmanuelEsuong/introductionToHtml/index.html [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+<html lang="en">
+<head>
+    <p> <img src="Image/Foodco upmata.PNG" controls></img></p>
```
Comment (ToluAkin) [inline submissions/emmanuelEsuong/introductionToHtml/index.html:4]: You can't add this elements in the head element.

File: submissions/emmanuelEsuong/introductionToHtml/index.html [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+</head>
+
+<body>
+    <!--Products ads animation-->
+    <p style="text-align:center;"><img src="Image/Foodco%20ads.gif" width="1550" height="600" controls></img> </p>
+    
+    <!--Menu list-->    
+    <table style="border-spacing:20px;margin-left:auto;margin-right:auto;">
+        <tr>
+            <td> <img src="Image/Foodco s.PNG" controls></img> </td>
```
Comment (ToluAkin) [inline submissions/emmanuelEsuong/introductionToHtml/index.html:41]: The images used in the file are not added to this pull request. Update that.

Comment (ToluAkin) [reply submissions/emmanuelEsuong/introductionToHtml/index.html:41]: You are meant to use only 2 product images. Check the criteria here.
https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/issues/7

File: submissions/emmanuelEsuong/introductionToHtml/index.html (hunk 2 above)
Comment (ElJentle) [reply submissions/emmanuelEsuong/introductionToHtml/index.html:4]: Thanks for the reviews. 

Please is it the paragraph (I. e  <p) or the image itself ?

File: submissions/emmanuelEsuong/introductionToHtml/index.html (hunk 1 above)
Comment (ElJentle) [reply submissions/emmanuelEsuong/introductionToHtml/index.html:1]: Thanks for the reviews. 

Please can you kindly explain more? Removing all styles...  I don't really get this.

File: submissions/emmanuelEsuong/introductionToHtml/index.html (hunk 2 above)
Comment (ToluAkin) [reply submissions/emmanuelEsuong/introductionToHtml/index.html:4]: Both

File: submissions/emmanuelEsuong/introductionToHtml/index.html (hunk 1 above)
Comment (ToluAkin) [reply submissions/emmanuelEsuong/introductionToHtml/index.html:1]: You are using inline styling in your work while the first acceptance criteria is `Develop a product page for your e-commerce store using HTML Only` emphasizes on ONLY.

**Pull request closed without merging**